content/proposals/apo.md

```diff
@@ -34,6 +38,8 @@ This proposal has been specified in
 
 ## Use Cases
 
-APO was specifically proposed to bring about Eltoo, but can be used to achieve various other use
-cases. However,
-
+APO was specifically proposed to bring about [Eltoo](/use-cases/eltoo), but can be used to achieve various other use
+cases, such as [Statechains](/use-cases/statechains) and [Spacechains](/use-cases/spacechains). It
+can also be used to inefficiently emulate a [CTV](/proposals/ctv)-like covenant and all its
+use-cases, but the efficiency loss makes these use cases much less compelling than the 3 proposals
```

Comment:
What do you mean with the "3 proposals above"?

Reply:
Eltoo, Spacechains and Statechains.

But yeah, this is weird.

I was just trying to imagine what you meant with the cryptic "However," that ended the text before. What were you going to say?

Reply:
Ah yeah I made of lot of this pages in a hurry, I think I was going to say that in order to use APO for other things like cong contr or stuff, it's actually pretty roundabout and inefficient. Also fragile cuz you need to keep signatures etc.